controllers/rollingupgrade_controller.go

```diff
@@ -299,28 +299,33 @@ func (r *RollingUpgradeReconciler) GetAutoScalingGroup(rollupName string) (*auto
 // SetStandby sets the autoscaling instance to standby mode.
 func (r *RollingUpgradeReconciler) SetStandby(ruObj *upgrademgrv1alpha1.RollingUpgrade, instanceID string) error {
 	r.info(ruObj, "Setting to stand-by", ruObj.Name, instanceID)
-	input := &autoscaling.EnterStandbyInput{
-		AutoScalingGroupName:           aws.String(ruObj.Spec.AsgName),
-		InstanceIds:                    aws.StringSlice([]string{instanceID}),
-		ShouldDecrementDesiredCapacity: aws.Bool(false),
-	}
 
 	asg, err := r.GetAutoScalingGroup(ruObj.Name)
 	if err != nil {
 		return err
 	}
 
-	instanceState, err := getGroupInstanceState(asg, instanceID)
+	instanceState, err := getInstanceStateInASG(asg, instanceID)
 	if err != nil {
 		r.info(ruObj, fmt.Sprintf("WARNING: %v", err))
 		return nil
 	}
 
+	if instanceState == autoscaling.LifecycleStateStandby {
```

Comment:
Isnt this covered by the next block `if !isInServiceLifecycleState(instanceState)` ?
Do you just want to avoid the log entry?

Reply:
Yes, I want to avoid log entry. If instance is already in standby, we should not log anything. Also, investigating issue when upgrade doesn't finish.

Reply:
Are you ok to remove logging completely then?

Reply:
@eytan-avisror Should the log message be removed completely? Would it affect debuggability?